includes/Classifai/Providers/OpenAI/ChatGPT.php

```diff
@@ -189,6 +226,19 @@ public function setup_fields_sections() {
 		$roles = get_editable_roles() ?? [];
 		$roles = array_combine( array_keys( $roles ), array_column( $roles, 'name' ) );
 
+		/**
+		 * Filter the allowed WordPress roles for ChatGTP
+		 *
+		 * @since x.x.x
+		 * @hook classifai_chatgpt_allowed_image_roles
```

Comment:
I don't think this filter should have `image` in it, I'm assuming this was probably just a copy/paste error

Reply:
@dkotter The fix for this is now ready for review. Thanks!

cc @jeffpaul